### PR TITLE
Updated node.js docs with instructions for using the 'files' section …

### PR DESCRIPTION
**Pull request by rhoads-zach**
…in package.json. Fixes RHOARDOC-1189

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@lholmquist let me know what you think

**Comment by lholmquist** [comment]
👍

File: docs/topics/proc_creating-nodejs-app.adoc [hunk 1]
```diff
@@ -102,6 +102,34 @@ $ npm install nodeshift --save
...
+  },
+  "files": {
+    "package.json",
+    "app.js"
+  }
+  ...
+}
+----
++
+The `files` section tells `nodeshift` what files to include when deploying to OpenShift. If the `files` section is not specified, `nodeshift` will send the entire current directory, excluding:
```
Comment (Ladicek) [inline docs/topics/proc_creating-nodejs-app.adoc]: Multiple sentences on single line :-)

Also, more importantly -- can the entries in the `files` section contain wildcards? I'd expect to find out in this description. @lholmquist?

Comment (lholmquist) [reply docs/topics/proc_creating-nodejs-app.adoc]: like how wild are you thinking.

if you want to do a directory, for example, `lib`, you can just specificy `lib` and that will get everything in that directory including sub directories.

this is the module we use to do that archiving: https://github.com/npm/node-tar
and we pass in an array of files to the create method.

if there is some crazy pattern matching stuff that someone needs, then i'd imaging they've written their app wrong

Comment (Ladicek) [reply docs/topics/proc_creating-nodejs-app.adoc]: I think that's perfectly enough, but from the present text, I didn't get the impression that I can include directory names as well. I actually got the opposite impression -- that I need to list all the files.

Comment (lholmquist) [reply docs/topics/proc_creating-nodejs-app.adoc]: that is true.  lets tell @rhoads-zach to add it  :)

**Comment by rhoads-zach** [comment]
@lholmquist @Ladicek updated per feedback.  Let me know what you think.

**Comment by lholmquist** [comment]
👍

**Comment by rhoads-zach** [comment]
thank you @lholmquist @Ladicek 

merging.

**Pull request merged**